cypress/e2e/12-canvas.cy.ts

```diff
@@ -0,0 +1,286 @@
+import { MANUAL_TRIGGER_NODE_NAME, CODE_NODE_NAME, SCHEDULE_TRIGGER_NODE_NAME, SET_NODE_NAME } from './../constants';
+import { WorkflowPage as WorkflowPageClass } from '../pages/workflow';
+
+const WorkflowPage = new WorkflowPageClass();
+
+const DEFAULT_ZOOM_FACTOR = 1;
+const ZOOM_IN_X1_FACTOR = 1.25; // Zoom in factor after one click
+const ZOOM_IN_X2_FACTOR = 1.5625; // Zoom in factor after two clicks
+const ZOOM_OUT_X1_FACTOR = 0.8;
+const ZOOM_OUT_X2_FACTOR = 0.64;
+const RENAME_NODE_NAME = 'Something else';
+
+describe('Canvas Actions', () => {
+	beforeEach(() => {
+		cy.resetAll();
+		cy.skipSetup();
+		WorkflowPage.actions.visit();
+		cy.waitForLoad();
```

Comment:
I think this was just a personal decision of the authors. I was adding this to my suites since I wanted to make sure new workflow page is fully loaded before I start doing anything.